src/sass/myservice-transaction-table.scss

```diff
@@ -128,7 +128,7 @@
   .transactions-table {
     border: 1px solid #d4d4d4;
     border-radius: 4px;
-    overflow: hidden;
+    // overflow: hidden;
```

Comment:
can you remove it instead of commenting it out? just a good practise to not leave commented code around without actual comments

Reply:
Yep done